packages/astro/src/runtime/client/deserialize.ts

```diff
@@ -0,0 +1,25 @@
+interface PropTypeSelector {
```

Comment:
I don't see that this file is being used anywhere. What's it for?

Reply:
Whoops, leftover from some earlier work.